scripts/Create-StorageContainer.ps1

```diff
@@ -71,59 +75,91 @@ trap [Exception]
     # this should trigger and catch and resume
     throw "Script execution failed: $($_)";
 }
+function New-Containers
```

Comment:
- [ ]  nit: new line.
- [ ]  You may want to move these new functions into common.ps1 and simplify the interface here.
- [ ]  You may want to fully qualify the function name. Something like New-AzureStorageContainers seems more appropriate. The same applies to the other 2 function being introduced.